### PR TITLE
Add new blogpost, new author image and header image.

### PR DESCRIPTION
**Pull request by ywolpers**

### CODE REVIEW

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+And we all want happy users, right?
+
+## How to make your product speak "local"
+
+### Step 1: Research
+
+First of all, figure out the scope. Not all regional wording is relevant for a software product. For us, we had to look out for words related to E-commerce, so we did not have to deep-dive into all those mouth-watering words for regional dishes (although we couldn't help but stumble upon "Palatschinken" and "Rösti"), but rather focused on words like "Kassa" (Austrian term for cashier or checkout) or "Total" (Swiss term for total sum).
+
+The research we did was mainly through online word lists and local websites, online shops and news sites. Wikipedia, for example, offers extensive lists of Helvetisms and Austriacisms that were very helpful as a starting point.
```
Comment (amahaiuddin) [inline _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:49]: Suggestion: Maybe you could link to the Helvetisms and Austriacisms lists.

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+### Step 2: Ask a native speaker to review
+
+Ok,  so you've done your research and came up with a list of local words that are relevant for your audience. But how do you make sure these are words that locals actually use? How do you avoid sounding as if you're trying too hard, which is the exact opposite of the smooth local experience you want to deliver to your users?
```
Comment (amahaiuddin) [inline _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:53]: Remove double space before "so": Ok,  so...

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+header_overlay: true
+category: language-and-localization
+tags: ["UI", "wording", "localization", "language", "translation"]
+authors: ["Yasmine"]
+about_authors: ["ywolpers"]
+---
+
+So your software has been translated into several languages already and as there are languages that are spoken in more than one country, you can conveniently cover several countries with just one language, right?
+
+But can you, really?
```
[the remark this answers is not in the document]
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:16]: @cgebken I think it works with the comma, what do you think? Does it, really? ;)

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+## Language varieties
+
+Each language has many different varieties, or forms of language. There are varieties for specific purposes, e.g. language that is being used for a public speech is different from language being spoken in a casual conversation.
+There are also varieties for specific linguistic styles: Teenagers, for example, speak a different variety of the same language than their parents.
+
+And then there's regional varieties.
```
[the remark this answers is not in the document]
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:23]: Fixed, thanks!

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+The differences between language varieties spoken in different regions of the same country or in different countries sharing the same language can range from subtle to "is that really still the same language"?
+
+So can you cover several countries with just one language? 
+
+## Yes and no
+
+Let's take German as an example. When localizing software for the German speaking market, this usually includes Germany, Austria and the German-speaking part of Switzerland and is typically referred to as the DACH region (DE-AT-CH).
```
[the remark this answers is not in the document]
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:30]: I don't think the comma is mandatory here, but it does make it easier to read, so I've added it in :)

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+**Yes**: German speakers of all three countries usually _understand_ Standard German.
+
+**No**: Standard German does not feel _local_ for speakers of Austrian Standard Germand and Swiss Standard German. It is similar to how speakers of American English feel towards British English. It may sound "cute" in some cases, "weird" in others or downright incomprehensible in others. 
+Spiffing, right?
+
+## Support your local varieties
+
+If you go the extra mile to make your software product sound local, it will come more natural for speakers of all regional varieties to use your product. They won't frown upon "strange expressions", may smile when being greeted in a very local way and will unconsciously feel better about your product.
```
[the remark this answers is not in the document]
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:39]: I double-checked, this sentence should be without comma I think.
image 
https://www.grammarly.com/blog/comma-before-and/

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md (hunk 1 above)
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:49]: Good idea, but I am not sure if we should add the links, as they're in German (there is an English version of the Helvetism one, but the word list in there is much shorter). What do you think?

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (amahaiuddin) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:16]: I'm fine with: But can you, really? :)

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (amahaiuddin) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:39]: 👍

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md (hunk 1 above)
Comment (amahaiuddin) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md:49]: Oh, right. I didn't think about it. I think then it's okay not to add the link.

File: _data/authors.yml [hunk 7]
```diff
@@ -650,4 +650,10 @@
...
   full_name: Xavier Alvarez
   description: is a Java and Kotlin Backend Developer, and an Ubuntu fanboy.
-  image: avatar_male.jpg
\ No newline at end of file
+  image: avatar_male.jpg
+
+- id: ywolpers
+  name: Yasmine
+  full_name: Yasmine W.
+  description: is a UX-Writer/Tech-Writer at ePages. She loves working with words and languages and is passionate about finding just the right wording. Even if that means deciding the original wording was actually the best option.
```
Comment (amahaiuddin) [inline _data/authors.yml:658]: I think it should be: UX Writer / TechWriter

File: _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md (hunk 1 above)
Comment (ywolpers) [reply _posts/2020/2020-07-30-how-to-localize-software-for-different-language-varieties.md]: To close the loop here - discussed in person, decided there is value in adding the links, so added them in after all.

**Pull request merged**